src/NuGet.Core/NuGet.CommandLine.XPlat/ReportRenderers/Enums/ReportOutputVersion.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal enum ReportOutputVersion
```

Comment:
Can we just use an int?
I don't think the enum improves readability.

Reply:
Sure, removed.